src/App.tsx

```diff
@@ -19,12 +84,30 @@
               placeholder="Enter a part of the name"
               className="input"
               data-cy="search-input"
+              value={query}
+              onChange={handleChangeInputName}
+              onFocus={handleFocus}
+              onBlur={handleBlur}
             />
           </div>
 
-          <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
-            <div className="dropdown-content">
-              <div className="dropdown-item" data-cy="suggestion-item">
+          {showArray && (
+            <div
+              className="dropdown-menu"
+              role="menu"
+              data-cy="suggestions-list"
+            >
+              <div className="dropdown-content">
+                {updatedPersons.map(people => (
+                  <div
+                    className="dropdown-item"
+                    data-cy="suggestion-item"
+                    key={people.name}
+                  >
+                    <p className="has-text-link">{people.name}</p>
+                  </div>
```

Comment:
When a suggestion is clicked, ensure the text is saved to the input, the list closes, and the selected person is passed to the `onSelected` callback. This functionality seems to be missing.